apparmor.d/abstractions/chromium

```diff
@@ -13,11 +13,15 @@
 # @{cache_dirs} = @{user_cache_dirs}/chromium
 
   include <abstractions/audio>
+  include <abstractions/bus-session>
+  include <abstractions/bus-system>
+  include <abstractions/bus/org.freedesktop.UPower>
   include <abstractions/dconf-write>
   include <abstractions/desktop>
   include <abstractions/devices-usb>
   include <abstractions/fontconfig-cache-read>
   include <abstractions/graphics-full>
+  include <abstractions/kde-strict>
```

Comment:
You already have `include <abstractions/desktop>` that is a generic version of `kde-strict` and `gnome-strict`